### PR TITLE
Chore: update local testing workflow & doc

### PR DESCRIPTION
**Pull request by tristanwagner**

### CODE REVIEW

**Comment by josermarinr** [comment]
Rebase and update typescript 
Screenshot 2022-09-14 at 14 56 45

**Comment by tristanwagner** [comment]
> 

done

File: yarn.lock [hunk 1]
```diff
@@ -3871,45 +3871,56 @@ __metadata:
...
-  version: 0.34.17
-  resolution: "@volar/code-gen@npm:0.34.17"
+"@volar/code-gen@npm:0.40.13":
+  version: 0.40.13
+  resolution: "@volar/code-gen@npm:0.40.13"
   dependencies:
-    "@volar/source-map": 0.34.17
-  checksum: 89cd9f4266bc2a83e9eaae80e50e9032d8cf95c867c507aa64291e51bf81d01e3054ba5a1eb254648ff9516b41076e44045d11f2d5711fd3fc18a3fc78d873b0
+    "@volar/source-map": 0.40.13
+  checksum: 70c1282d72c33b27b3c08f16f1de6888b66226ce1cf79bfa2e43c3652da399f68847e026ba83293044376d1a8470289dd7dc23c17feb6e75c1cba3e2787817d1
```
Comment (josermarinr) [inline yarn.lock:3879]: why we have change is this dependency?

File: yarn.lock [hunk 2]
```diff
@@ -17316,16 +17422,17 @@ __metadata:
...
   dependencies:
-    "@volar/vue-typescript": 0.34.17
+    "@volar/vue-language-core": 0.40.13
+    "@volar/vue-typescript": 0.40.13
   peerDependencies:
     typescript: "*"
   bin:
     vue-tsc: bin/vue-tsc.js
-  checksum: 0c5addb114b6e4d20e131ebdc95806fc7b723f2301735b929f3e9e70491787f13b0318b25b75af50e32c6639754b62c62182483a1eda0fa63448436f3e82a505
+  checksum: 00e5cb424f1ab1a198489400b165a1d3968f85bf2747fe1cc8d8ac7161752a407d68e703f0132a547eaa8b236d1454627b92771cf268a968e5ac191b18de9690
```
Comment (josermarinr) [inline yarn.lock:17435]: this one I know but the other change I dont catch why if the dependencies are the same

**Review by josermarinr — APPROVED** [review]

**Review by josermarinr — APPROVED** [review]

**Pull request merged**